lib/src/container/store.rs

```diff
@@ -24,6 +25,10 @@ const IMAGE_PREFIX: &str = "ostree/container/image";
 const META_MANIFEST_DIGEST: &str = "ostree.manifest-digest";
 /// The key injected into the merge commit with the manifest serialized as JSON.
 const META_MANIFEST: &str = "ostree.manifest";
+/// Value of type `a{sa{su}}` containing number of filtered out files
+pub const META_FILTERED: &str = "ostree.tar-filtered";
+/// The type used to store content filtering information with `META_FILTERED`.
+pub type MetaFilteredData = HashMap<String, HashMap<String, u32>>;
```

Comment:
Is this useful this way though? Wouldn't a consumer prefer a map of `layer_id` -> `set of filtered paths`?

Reply:
This data is stored once per commit, and we have a one-one mapping of layer to commit.

Reply:
We could add an API that loaded all the commits and printed that I guess too.

Reply:
Ack, though I'm not sure what to take away from that statement. Are you saying that it would be too much data to store?

For clarity, I'm not concerned with the keys of this map, but with the values: they seem more obscure and less useful for the typical operation that a consumer would do (i.e. list/query/print entries for filtered paths).

In pseudo-types, my original comment was asking: why not doing `HashMap<LayerDigestString, Vec<FilteredPathString>>` instead?

Reply:
Yeah, need to document this better but basically the map is e.g.:

- `/var` -> 42
- `/boot` -> 3

In order to provide some information about filtered paths, while also not having metadata that scales linearly with the content size.

Reply:
> not having metadata that scales linearly with the content size

Ok thanks, that answered it.